gradle4/tests/scan.bash

```diff
@@ -0,0 +1,10 @@
+  echo "buildScan {" >> "/hab/cache/src/hello-gradle/build.gradle"
```

Comment:
Same as comment for `gradle/tests/scan.bash` - Don't use bash, just include the file.